club/theme.json

```
@@ -37,17 +37,27 @@
                 }
             ],
             "palette": [
-				{
-					"slug": "foreground",
-					"color": "#000AFF",
-					"name": "Foreground"
-				},
-				{
-					"slug": "background",
-					"color": "#F6EB80",
-					"name": "Background"
-				}
-			]
+                {
+                    "slug": "foreground",
+                    "color": "#000AFF",
+                    "name": "Foreground"
+                },
+                {
+                    "slug": "background",
+                    "color": "#F6EB80",
+                    "name": "Background"
+                },
+                {
+                    "slug": "primary",
+                    "color": "#000AFF",
+                    "name": "Primary"
+                },
+                {
+                    "slug": "tetriary",
+                    "color": "#FFF7A7",
+                    "name": "Tetriary"
+                }
```

👋  nothing to add other than this small typo:

```suggestion
                {
                    "slug": "tertiary",
                    "color": "#FFF7A7",
                    "name": "Tertiary"
                }
```